source/PreferencesPanel.cpp

```diff
@@ -438,11 +438,12 @@ void PreferencesPanel::DrawSettings()
 		"Show hyperspace flash",
 		"\n",
 		"Other",
+		"Interactive radar display",
```

Comment:
I feel like "*Clickable* radar display" would make the functionality more obvious than "Interactive."

Reply:
:grinning: I had almost changed it to that when merging yesterday, and then second guessed myself because clickable didn't relay back that the game would respond to the user clicking it.